manifests/init.pp

```diff
@@ -46,7 +46,10 @@
   $url,
   $destination_directory,
   $destination_file = '',
-  $proxyAddress=''
+  $proxyAddress='',
+  $proxyUser='',
+  $proxyPassword='',
+  $isPasswordSecure=true
```

Comment:
I agree from a usability stance that false would be better. One thing to note here is that I believe the secure string can only be created on the node itself... so not sure how you would get the secure string for an arbitrary node. For me, I was doing masterless, so it made sense. For most it probably won't